pkg/lib/cockpit.d.ts

```diff
@@ -121,10 +121,22 @@ declare module 'cockpit' {
         close(options?: JsonObject): void;
     }
 
-    interface ChannelOptions {
-        payload: string;
+    // these apply to all channels
+    interface BaseChannelOptions {
```

Comment:
I think this one should have still been called `ChannelOptions`...

Reply:
Done.